config/hp.yml

```diff
@@ -19,28 +19,24 @@ entries:
 - prefix: /tracker
   replacement: https://github.com/obophenotype/human-phenotype-ontology/issues
 
+- prefix: /releases/hpoa/
+  replacement: https://github.com/obophenotype/human-phenotype-ontology/releases/
```

Comment:
This says: if you see a PURL such as:

http://purl.obolibrary.org/obo/hp/releases/hpoa/2020-02-14/genes_to_phenotype.txt

redirect to the respective github release.